### PR TITLE
Added content-type admin API responses

### LINKED ISSUE #1745: bug: Incorrect content-type header in Admin API Responses
### Issue description
APISIX Admin API sets the content-type as text/plain when the actual content-type is application/json

### Environment

* apisix version (cmd: `apisix version`): master
* OS: ubuntu

### Minimal test code / Steps to reproduce the issue
1. Send any valid request to APISIX Admin API
2. Response headers contain content-type as text/plain

### What's the actual result? (including assertion message & call stack if applicable)

Response Headers contain `Content-Type: text/plain`

### What's the expected result?

Response Headers should contain `Content-Type: application/json`

### PR DESCRIPTION
**Pull request by shenal**
### Summary

SUMMARY_HERE
This PR adds `Content-Type: application/json` to the Admin API response headers fixing this [issue](https://github.com/apache/incubator-apisix/issues/1745)

### Issues resolved 

Fix #1745

### CODE REVIEW

**Comment by membphis** [comment]
please add some test cases about this.

**Comment by membphis** [comment]
@shenal pls take a look at the output of Travis-CI:

```
Test Summary Report
-------------------
t/admin/routes.t                     (Wstat: 1024 Tests: 150 Failed: 4)
  Failed tests:  2, 15, 19, 23
  Non-zero exit status: 4
Files=116, Tests=3847, 882 wallclock secs ( 1.13 usr  0.12 sys + 120.55 cusr 21.98 csys = 143.78 CPU)
Result: FAIL
The command "$PWD/.travis/${OSNAME}_runner.sh script" exited with 1.
```

**Comment by membphis** [comment]
@shenal I suggest not to modify any existing test cases, you can add new ones for testing.

**Comment by shenal** [comment]
> @shenal I suggest not to modify any existing test cases, you can add new ones for testing.
> This will make your PR simpler.

@membphis Noted. I was investigating the test suite and found out that the test client defined in `test_admin.lua` does not return the response headers for me to assert. I am looking for a different approach

File: t/admin/routes.t [hunk 1]
```diff
@@ -321,6 +328,7 @@ GET /t
                 }]]
                 )
 
+            ngx.header["Content Type"] = res.header["Content Type"]
```
Comment (juzhiyuan) [inline t/admin/routes.t:331]: image

**Comment by membphis** [comment]
@shenal if you need any help, please let us know. ^_^

**Comment by shenal** [comment]
> @shenal if you need any help, please let us know. ^_^

Thanks @membphis  appreciate it :smiley: . This issue made me dig more into the internals of the `lua-resty-http` client. I think the tests should pass now :crossed_fingers:

File: t/admin/routes.t (hunk 1 above)
Comment (shenal) [reply t/admin/routes.t]: I think this issue has been already resolved in Openresty

**Comment by shenal** [comment]
@membphis FYI The tests have passed. Third time was the charm

**Comment by membphis** [comment]
@shenal many thx

**Pull request merged**